uv.lock

```diff
@@ -1441,7 +1441,7 @@ provides-extras = ["touchstone"]
 
 [[package]]
 name = "mitol-django-common"
-version = "2025.6.20"
+version = "2025.8.1"
```

Comment:
For some reason, the release command does not update uv.lock when creating a new release so the new release tag is updated in new PRs. We can keep it here until we fix the release command to add this change as part of release.